App/Services/utils.js

```diff
@@ -0,0 +1,10 @@
+export const coercePhoneNumber = phoneNumber => {
+  if (phoneNumber.length < 10) return new Error('Phone Number not long enough')
```

Comment:
Also, throwing an error instead of creating one causes the whole app to fall over.